src/bower/DependenciesManager.js

```diff
@@ -77,10 +77,41 @@ define(function (require, exports) {
         return FileUtils.exists(path);
     }
 
+    /**
+     * Get the dir of the dependencies
+     * @param {string=} path
+     * @return {Promise}
+     */
+    function findBowerJsonLocation(path) {
+        if (!path) {
+            path = ProjectManager.getProjectRoot().fullPath;
+        }
+
+        return path;
+    }
+
     function _loadBowerJson(path) {
         _bowerJsonFile = new BowerJsonFile(path);
     }
 
+    function getDependencies() {
+        var deferred = new $.Deferred();
+
+        _bowerJsonFile.getContent()
+            .done(function (data) {
+                var dependencies = [];
+                data = JSON.parse(data);
+
+                _.each( data.dependencies, function (version, name) {
+                    dependencies.push({name:name, version:version});
+                } );
+
```

Comment:
This should be placed in getDependencies of BowerFile,since it the BowerFile responsibility on how to parse all of this information.
DependenciesManager will handle multiple instances of BowerFile according to the current working directory, of course only one will be the active one, but in the creation time this will handle that. And maybe listing all of the available bower.json files..